yaml/pois.yaml

```diff
@@ -946,7 +946,15 @@ filters:
   ############################################################
   # parking
   - filter: {amenity: parking}
-    min_zoom: 17
+    min_zoom:
+      lookup:
+        key:  { call: { func: tz_estimate_parking_capacity, args: [ { col: capacity }, { col: 'parking' }, { col: 'building:levels' }, { col: 'way_area' } ] } }
+        op: '>='
+        table:
+          - [ 15, 350 ]
```

Comment:
Perhaps this should be even bigger now that we have the capacity measurements?

```
           - [ 15, 1000 ]    # was 350
           - [ 16,   200 ]    # was 130
           - [ 17,      45 ]    # no change, seems fine
```

Looking at Disneyland in California:
17/33.80043/-117.91088

![image](https://user-images.githubusercontent.com/853051/50529763-23a24300-0aac-11e9-8a66-7d4b6eb67830.png)

There are also some absurdly large parking garages (maybe OSM tagging issues?) in Los Vegas on the Strip:
15/36.1038/-115.1634

But I think still zoom 15 is good first zoom for those (and likely client side don't show the name).

![image](https://user-images.githubusercontent.com/853051/50529863-f1451580-0aac-11e9-8b04-a628b52da807.png)

Reply:
The parking garages in Los Vegas are tagged correctly, our building logic just isn't exporting parking_garage as the kind_detail correctly.